msg/CollisionObject.msg

```diff
@@ -1,27 +1,36 @@
-# a header, used for interpreting the poses
+# A header, used for interpreting the poses
 Header header
 
-# the id of the object (name used in MoveIt)
+# The id of the object (name used in MoveIt)
 string id
 
 # The object type in a database of known objects
 object_recognition_msgs/ObjectType type
 
-# the the collision geometries associated with the object;
-# their poses are with respect to the specified header
+# The collision geometries associated with the object.
+# Their poses are with respect to the specified header
 
-# solid geometric primitives
+# Solid geometric primitives
 shape_msgs/SolidPrimitive[] primitives
 geometry_msgs/Pose[] primitive_poses
 
-# meshes
+# Meshes
 shape_msgs/Mesh[] meshes
 geometry_msgs/Pose[] mesh_poses
 
-# bounding planes (equation is specified, but the plane can be oriented using an additional pose)
+# Bounding planes (equation is specified, but the plane can be oriented using an additional pose)
 shape_msgs/Plane[] planes
 geometry_msgs/Pose[] plane_poses
 
+# Named frames on the object. Use these to define points of interest on the object that you want
+# to plan with (e.g. screwdriver_tip, kettle_spout, mug_base).
+# The length of these vectors has to be identical.
+# The names of named frames in the scene need to be unique! If both "screwdriver" and "pencil" have
```

Comment:
You're probably right. I think it would already work to just use underscore separators like bottle_1 bottle_2 -> bottle_1_tip / bottle_2_tip

Reply:
I also thought of underscores a few hours after writing that. Probably better than introducing slashes. I'm in favor of adding this change, but I don't want to overload this PR. Will be happy to work on this afterwards though, because I will need that functionality as well.

Reply:
Slashes are now the separator. This allows quick checking if a subframe is part of the frame_name.